### DISCUSSION TITLE
是否可以提供电商相关的插件实现商品订单支付等功能类似wordpress 一样可

### DISCUSSION THREAD

**Discussion by fireflyhoo** [halo-dev/halo#4751] [Ideas]
### 你当前使用的版本

1.3.0

### 描述一下此特性

是否可以提供电商相关的插件实现商品订单支付等功能类似wordpress 一样可
最近想弄个跨境电商的网站, 我觉得可以基于 halo来搭建,但是看下来没有相关的插件

**Comment by ruibaby** [comment]
/kind feature
/area plugin
/priority awaiting-more-evidence

如果要通过插件来实现一套完整的跨境电商系统，工作量可能会非常大。目前建议使用成熟的电商系统或者 SaaS 平台，Halo 对于这种使用场景会持续关注。

**Comment by fireflyhoo** [comment]
wordpress 有类似的电商插件, 基于 wordpress 是可以做的,但是考虑到 wordpress 是php开发的维护起来略麻烦, 如果halo有类似的插件的话, 应用会广很多.
另外就是halo 后续是否提供 类似 插件市场类似的东西, 可以在线安装插件 如idea 一样. 当前安全插件的步骤略微繁琐,非IT人员使用起来有一定的困难

**Comment by ruibaby** [comment]
> 另外就是halo 后续是否提供 类似 插件市场类似的东西, 可以在线安装插件 如idea 一样. 当前安全插件的步骤略微繁琐,非IT人员使用起来有一定的困难

目前 Halo 的官网已经提供了应用市场的功能：https://halo.run/store/apps

后续是有计划将应用市场集成到 Halo 的控制台的。此外，应用市场后续应该也会做付费的功能，所以这也算是小型电商场景的一种尝试。

**Comment by ruibaby** [comment]
我将把此 issue 转移到 Discussion 的 Ideas 分区。
